config/m4/cuda.m4

```diff
@@ -74,6 +75,12 @@ AS_IF([test "x$cuda_checked" != "xyes"],
                               have_cuda_static="yes"],
                              [], [-ldl -lrt -lpthread])])
 
+         AC_CHECK_DECLS([CU_MEM_HANDLE_TYPE_FABRIC],
+                        [have_cuda_fabric=yes], [], [[#include <cuda.h>]])
+
+         AS_IF([test "x$have_cuda_fabric" = "xyes"],
+               [AC_DEFINE([HAVE_CUDA_FABRIC], 1, [Enable CUDA fabric handle support])], [])
+
```

Comment:
It seems that it can be simplified.
```
         AC_CHECK_DECLS([CU_MEM_HANDLE_TYPE_FABRIC],
                        [AC_DEFINE([HAVE_CUDA_FABRIC], 1, [Enable CUDA fabric handle support])],
                        [], [[#include <cuda.h>]])
```
, and `have_cuda_fabric` can be removed.